pr_check.sh

```diff
@@ -13,12 +13,12 @@ IQE_FILTER_EXPRESSION=""
 
 # Install bonfire repo/initialize
 # CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
-# curl -s $CICD_URL/bootstrap.sh -o bootstrap.sh
-# source bootstrap.sh  # checks out bonfire and changes to "cicd" dir...
+# curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
```

Comment:
@gburges what's the reason for this change? 
If I omit all of this is commented, the bootstrap.sh is doing `cd $CICD_ROOT`